maintainer/certs.crt

```diff
@@ -0,0 +1,40 @@
+-----BEGIN CERTIFICATE-----
```

Comment:
Should this file be here?

Reply:
I have no idea what this is, so probably not?

Reply:
I believe @lilyminium added it, I'll let her make the call here :)

Reply:
Sorry I should have deleted it, I added it as part of trying to fix ca-certificates